### DISCUSSION TITLE
第3节课作业(8班)

### DISCUSSION THREAD

**Discussion by Hongru0306** [InternLM/Tutorial#172] [作业]
请在下方提交作业链接,格式为
-->报名序号+连接:
例如:213+https://kvudif1helh.feishu.cn/docx/...

**Comment by BJUT-WangZhengTao** [comment]
371+https://blog.csdn.net/m0_54169072/article/details/135433926

**Comment by baijiesong** [comment]
509+https://github.com/baijiesong/InternLM_Learning/blob/main/%E7%AC%AC%E4%B8%89%E8%8A%82%E4%BD%9C%E4%B8%9A.md

**Comment by Yujin320** [comment]
587+https://github.com/Yujin320/InternLM-hyx-Learning/blob/main/homework03.md

**Comment by L-369-maha** [comment]
521+https://github.com/L-369-maha/InternLM/blob/main/Three-LangChain/Task.md

**Comment by superkong001** [comment]
545+https://github.com/superkong001/InternLM_Learning/blob/main/homework2.md

**Comment by NYYY0421** [comment]
501+https://github.com/NYYY0421/InternLM-langchain/blob/main/homework3_langchain.md

**Comment by Alias-z** [comment]
388+https://github.com/Alias-z/InternLM-Camp-2024/tree/main/Assignment%202

**Reply by Alias-z** [reply]
更新一下链接，文件夹改名了 https://github.com/Alias-z/InternLM-Camp-2024/tree/main/Assignment%202%20LangChain

**Comment by Cyydz** [comment]
527+https://www.wolai.com/3oPwED5Ky7L7a9Cfa2vCA6

**Comment by ZPfree** [comment]
605+https://github.com/ZPfree/homework/blob/main/3homewordbase.md

**Comment by zhaoxin111** [comment]
445+https://app.yinxiang.com/fx/e1234006-2158-4fad-b596-b14cfb8d403c

**Comment by hzsun1995** [comment]
446+https://github.com/hzsun1995/internlm-course/blob/main/course3_hw.md

**Comment by JieGenius** [comment]
555+https://github.com/JieGenius/InternLMLearning/blob/main/lesson3/assignment.md

**Comment by jamiechoi1995** [comment]
554+https://blog.csdn.net/sinat_26253653/article/details/135563160

**Comment by carlinl** [comment]
596 + https://pt35a5ibej.feishu.cn/docx/JC1cd4vxRoF1pTxL1tkcFqKuneg?from=from_copylink

**Comment by Doublefei-Giser** [comment]
409+https://blog.51cto.com/u_16507376/9240214

**Comment by sunxiaobei** [comment]
585+https://github.com/sunxiaobei/InternLM_tutorial/blob/main/notes/HWDay03.md
